other/install_2-1_mysql.sql

```diff
@@ -353,7 +354,7 @@ CREATE TABLE {$db_prefix}log_errors (
 #
 
 CREATE TABLE {$db_prefix}log_floodcontrol (
-  ip VARBINARY(16),
+  ip VARBINARY(16) NOT NULL,
```

Comment:
a null able field is by sql def default null -> so no need to write this down.

Reply:
Note on floodcontrol, this was made NOT NULL to be consistent with the logic that rebuilds it in the upgrader.

Reply:
varbinary(16) = ip_field >needs to be null able<, not null is not allowed in any ip field.

Reply:
To get the PR thru, I'll make this change....

Reply:
For the record, since ip is in the primary key, NOT NULL is implicit - it will be enforced by the db engine.  
What this means is that removing NOT NULL will result in no change.